rbx_binary/src/deserializer/state.rs

```diff
@@ -1174,18 +1174,30 @@ rbx-dom may require changes to fully support this property. Please open an issue
                 VariantType::PhysicalProperties => {
                     for referent in &type_info.referents {
                         let instance = self.instances_by_ref.get_mut(referent).unwrap();
-                        let value = if chunk.read_u8()? == 1 {
-                            Variant::PhysicalProperties(PhysicalProperties::Custom(
-                                CustomPhysicalProperties {
-                                    density: chunk.read_le_f32()?,
-                                    friction: chunk.read_le_f32()?,
-                                    elasticity: chunk.read_le_f32()?,
-                                    friction_weight: chunk.read_le_f32()?,
-                                    elasticity_weight: chunk.read_le_f32()?,
-                                },
-                            ))
-                        } else {
-                            Variant::PhysicalProperties(PhysicalProperties::Default)
+                        let discriminator = chunk.read_u8()?;
+                        let value = match discriminator {
+                            0b00 | 0b10 => Variant::PhysicalProperties(PhysicalProperties::Default),
```

Comment:
Does this match Roblox's implementation (i.e. both the custom physical properties and acoustic absorption fields must be set to deserialize a custom acoustic absorption)?

Reply:
Yeah, Roblox is essentially using the acoustic absorption field as a "version" field to indicate whether a value supports `AcousticAbsorption` at all. It's why it's set even for new "not custom" values.